sshd-core/src/main/java/org/apache/sshd/common/util/io/IoUtils.java

```diff
@@ -67,6 +69,7 @@
     private static final byte[] EOL_BYTES = EOL.getBytes(StandardCharsets.UTF_8);
 
     private static final LinkOption[] NO_FOLLOW_OPTIONS = new LinkOption[]{LinkOption.NOFOLLOW_LINKS};
+    private static final AtomicReference<String> CURRENT_USER_HOLDER = new AtomicReference<>();
 
```

Comment:
No need for it if we agree that _System.getProperty("user.name")_ is the value we seek